dynamic-layers/qt5-layer/recipes-qt/qt5/qtbase_%.bbappend

```diff
@@ -5,6 +5,13 @@
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
+IMX_BACKEND = \
+    "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'wayland',\
+        bb.utils.contains('DISTRO_FEATURES',     'x11',     'x11', \
+                                                             'fb', d), d)}"
+SRC_URI_append = " \
```

Comment:
Wouldn't it make sense to only fetch this for imxgpu?
e.g. `SRC_URI:append:imxgpu = " \`
as the file is only used if the imxgpu override is present.

Reply:
Yes, for sure.